app/index.html

```diff
@@ -36,7 +36,8 @@
       <a href="contact.html">Contact</a>
     </nav>
 
-    <h1 class="title">Clio + Calliope</h1>
+    <img src="./images/logo_white.svg" alt="React JS logo" height="auto" width="80%" style="max-width: 960px;margin: 36px 0;"
```

Comment:
Looks like the alt text didn't get adjusted here from the initial copy pasta. We probably want `alt="Clio + Calliope"`. Is there a good reason to inline the styles here?

Reply:
Good catch! I will fix the alt. SVG's don't respond to anything but inline styles for sizing if used in an img tag. But also for page speed, Google recommends inline style for above-the-fold content so was thinking of adding some more in the future, depending on total amount.